### ISSUE TITLE
Require ability to set output filename consistent with other @easy-webpack/config-XXX

### ISSUE THREAD

**Issue by don-bluelinegrid** [easy-webpack/config-fonts-and-images#4]
In @easy-webpack/config-css, we have the ability to set the name or name format for the output file, based on this function signature:

```
export = function css({ filename = '[name].css', allChunks = false, sourceMap = false, extractText = undefined, resolveRelativeUrl = undefined, additionalLoaders = [], test = /\.css$/i } = {}) {
```
So, we can override the default name, and provide either a static name like ```vendor.css``` or a name format like ```[name]-[chunkhash].css```.

We should have this same capability in all configs that use loaders to output files, including this fonts-and-images config.  So, specifically, we should have the ability to pass in an 'options' object, such as this - ```{name:'[name]-[hash].[ext]'}``` - and see an output file such as this - ```fontawesome-webfont-c8ddf1e5e5bf3682bc7bebf30f394148.woff```.

This seems like a basic use case and requirement for loader configs, and since the @easy-webpack/config-css project offers this capability, this seems like an inconsistency in the implementations of the various config projects.

Thanks,
Don

**Comment by niieani** [comment]
Indeed, according to file-loader, we can set out own [filename templates](https://github.com/webpack/file-loader#filename-templates). I'd be happy with a pull request with the requested change implemented, as I'm quite busy recently, so I'm not sure when I'd be able to tackle this.

Best,
Bazyli
